### ISSUE TITLE
Allow textures to be scaled

### ISSUE THREAD

**Issue by Azhrei** [RPTools/maptool#259]
Allow textures to be scaled.  Perhaps the chooser panel that could have a slider to change the scaling before the texture is chosen?

The DrawExplorer would need some way to both rest the scale to 1:1 as well as change the scale.  Since background textures are drawing objects and not Tokens, how would scaling be stored?  Does the drawing object have a property for that?  If not, how much work would it be to add?

**Comment by rkathey** [comment]
Can we amend this to allow for rotation and transparency as well?

**Comment by Azhrei** [comment]
Yeah, those are both good additions as well.  I'm hoping @Jaggeroth might jump on this one, if he has time.

**Comment by rkathey** [comment]
Seems similar to #296
